contrib/plugins/cowsay.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/sh
+
+maybe_cowsay()
+{
+    cowsay || cat <<EOF
+ _________________________________
+< Please install 'cowsay' program >
+ ---------------------------------
+        \\   ^__^
+         \\  (xx)\\_______
+            (__)\\       )\\/\\
+             U  ||----w |
+                ||     ||
+EOF
+}
+
+# Eg. {"jsonrpc":"2.0","id":2,"method":"getmanifest","params":{}}\n\n
+read -r JSON
+read _
```

Comment:
Nit: `read -r _` to avoid mangling backslashes. Applies to all `read _` :-)

Reply:
Since we're not using `_` and it's assumed to be a blank line, this seems like the shortest code wins?

Reply:
.... but make check-source complains, so I changed it anyway!  Thanks!